### PR TITLE
content: translate pages

### LINKED ISSUE #29: Transcript pages for spanish site
- [x] Home @JeanVegaD 
- [x] Services @AngeloCG97 
- [x] Industries @JeanVegaD 
- [x] Team @JeanVegaD 
- [x] The company @JeanVegaD 
- [x] Block producers @AngeloCG97 
- [x] Press @AngeloCG97 
- [x] Projects @JeanVegaD 
- [ ] Blog @AngeloCG97 
- [x] Contact @JeanVegaD 
- [x] Aditonal resource @JeanVegaD 
- [x] Navbar @JeanVegaD 
- [x] Footer @AngeloCG97 
- [x] Suscribe form
- [ ] Do you need blockchain @JeanVegaD 
- [x] ContactUs Banner  @JeanVegaD
- [x] We are one group section @JeanVegaD

### PR DESCRIPTION
**Pull request by JeanVegaD**
closes #29 

#What does this PR do? 
- fix pahts
- fix navbar
- translate pages to spanish
  - Home 
  - Industries 
  - The company
  - Projects 
  - Contact 
  - Navbar 
  -  ContactUs Banner
  -  We are one group section
  -  Aditonal resource

### CODE REVIEW

File: src/pages/components/AdditionalResources/index.js [hunk 1]
```diff
@@ -11,47 +11,46 @@ const AdditionalResources = () => {
     <Box className="containerSec">
       <Box className="section">
         <Box className="titleBox">
-          <h2>Additional Resources</h2>
+          <h2>Recursos adicionales</h2>
```
Comment (xavier506) [inline src/pages/components/AdditionalResources/index.js:14]: >Recursos Adicionales (Title Case)

File: src/pages/components/AdditionalResources/index.js [hunk 2]
```diff
@@ -11,47 +11,46 @@ const AdditionalResources = () => {
...
               </Box>
             </Grid>
             <Grid onClick = {() => window.open('https://guide.eoscostarica.io/')} className="gridButton" item xs={12} md={5}>
               <Box className={clsx("centerBox","spacingBox")}>
                 <img className="resourcesIcon" srcSet={useBaseUrl("img/icons/decs.svg")}/>
                 <Box style={{marginLeft:'25px'}}>
-                  <h3 style={{marginBottom:'5px'}}>Devs</h3>
-                  <h3 style={{marginBottom:'5px'}}>Portal</h3>
+                  <h3 style={{marginBottom:'5px', width:'150px'}}>Portal de </h3>
+                  <h3 style={{marginBottom:'5px', width:'150px'}}>desarrollo</h3>
```
Comment (xavier506) [inline src/pages/components/AdditionalResources/index.js:44]: Portal de Desarrolladores

File: src/pages/components/AdditionalResources/index.js [hunk 3]
```diff
@@ -11,47 +11,46 @@ const AdditionalResources = () => {
...
                 </Box>
               </Box>
             </Grid>
             <Grid onClick={() => window.open('https://github.com/eoscostarica')} className="gridButton" item xs={12} md={5}>
             <Box className={clsx("centerBox","spacingBox")}>
                 <img className="resourcesIcon" srcSet={useBaseUrl("img/icons/github-ours.svg")}/>
                 <Box style={{marginLeft:'25px'}}>
-                  <h3 style={{marginBottom:'5px'}}>Our</h3>
-                  <h3 style={{marginBottom:'5px'}}>Github</h3>
+                  <h3 style={{marginBottom:'5px', width:'150px'}}>Nuestros</h3>
```
Comment (xavier506) [inline src/pages/components/AdditionalResources/index.js:52]: Nuestro GitHub

File: src/pages/components/ContactUsBanner/index.js [hunk 4]
```diff
@@ -12,12 +12,12 @@ const ContactUsBanner = () => {
                 <Grid container spacing={5}>
                     <Grid item xs={12} md={8}> 
                         <Box>
-                            <h2 className="centerTextOnMobile">Start working with us <br/> and implement <br/> blockchain technology. </h2>
+                            <h2 className="centerTextOnMobile">Empiece a trabajar con nosotros <br/> e implementar <br/> tecnología blockchain </h2>
```
Comment (xavier506) [inline src/pages/components/ContactUsBanner/index.js:15]: Comience a trabajar con nosotros implementando tecnología blockchain

File: src/pages/components/AdditionalResources/index.js (hunk 1 above)
Comment (ldrojas) [reply src/pages/components/AdditionalResources/index.js:14]: @xavier506 not necessary in Spanish to add title cases

**Review by AngeloCG97 — APPROVED** [review]

**Pull request merged**